source/ios/AdaptiveCards/ADCIOSFramework/ADCIOSFramework/ACRPresentationViewController.m

```diff
@@ -0,0 +1,44 @@
+//
+//  ACRPresentationViewController.m
+//  ADCIOS
+//
+//  Created by jwoo on 5/31/17.
+//  Copyright © 2017 Microsoft. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "ACRPresentationViewController.h"
+#import "ACRViewController.h"
+
+@implementation ACRPresentationViewController
+
+- (id) init:(NSString*) str{
```

Comment:
thanks

---
In reply to: [122553987](https://github.com/Microsoft/AdaptiveCards/pull/423#discussion_r122553987) [](ancestors = 122553987)